python/pyspark/pandas/plot/core.py

```diff
@@ -20,7 +20,7 @@
 import pandas as pd
 import numpy as np
 from pyspark.ml.feature import Bucketizer
-from pyspark.mllib.stat import KernelDensity
+from pyspark.mllib.stat.KernelDensity import KernelDensity
```

Comment:
Can you explain this change? It looks weird. Is it a nested class?

Reply:
Otherwise it raises `mypy checks failed:
python/pyspark/pandas/plot/core.py:23: error: Name 'KernelDensity' already defined (by an import)` for some reason.

Anyway I think it's really weird because seems like my changes don't affect to the `KernelDensity` side at all, but the `mypy` check started to fail.

And the more weird fact is that the `KernelDensity` is only imported here across all project, but it suddenly says already defined.

Let me just ignore this case for now, and investigate more in mypy side. I couldn't find the related bug so far.